twilight-gateway/src/shard.rs

```diff
@@ -151,10 +113,7 @@ pub enum ConnectionStatus {
     ///
     /// [failed authentication]: CloseCode::AuthenticationFailed
     /// [invalid intents]: CloseCode::InvalidIntents
-    FatallyClosed {
-        /// Close code of the close message.
-        close_code: CloseCode,
-    },
+    FatallyClosed,
```

Comment:
documentation needs to be changed if you remove the `close_code` here

Reply:
FatallyClosed is set by receiving a close code so I think it's good to mention that.